demos/serverless/deploy.js

```diff
@@ -26,7 +27,8 @@ function usage() {
   console.log(`  -e, --event-bridge                   Enable EventBridge integration, default is no integration`);
   console.log(`  -c, --chime-endpoint                 AWS SDK Chime endpoint, default is '${chimeEndpoint}'`);
   console.log(`  -t, --enable-termination-protection  Enable termination protection for the Cloudformation stack, default is false`);
-  console.log(`  -h, --help           Show help and exit`);
+  console.log(`  -l, --logs                           Disables printing logs`);
```

Comment:
- `--logs` => `--disable-printing-logs` to match the case below
- nit: `Disables` => `Disable` to be consistent with other messages

Reply:
Updated.

Reply:
The latest revision still has `--logs` over `--disable-printing-logs`